### PR TITLE
Removes air from mechtransport space ruin

### PR DESCRIPTION
**Pull request by coiax**
- Also replaces two computer frames with shuttle
  computer and exosuit control console.

### CODE REVIEW

**Comment by xxalpha** [comment]
:-1:

Use airless subtypes.

**Comment by Bawhoppen** [comment]
wait did you add docking ports if you're adding a shuttle console

**Comment by coiax** [comment]
Nah, it's a shuttle console that doesn't do anything. There's precedence for "former shuttles" to have a useless shuttle console, see free golem ship

File: _maps/RandomRuins/SpaceRuins/mechtransport.dmm [hunk 1]
```diff
@@ -1,65 +1,65 @@
...
-"y" = (/obj/effect/decal/cleanable/dirt,/turf/open/floor/plasteel/shuttle/yellow,/area/ruin/powered)
-"z" = (/obj/structure/mecha_wreckage/ripley,/turf/open/floor/plasteel/shuttle/yellow,/area/ruin/powered)
-"A" = (/obj/mecha/working/ripley,/turf/open/floor/plasteel/shuttle/yellow,/area/ruin/powered)
-"B" = (/obj/effect/decal/cleanable/oil,/turf/open/floor/plasteel/shuttle/yellow,/area/ruin/powered)
-"C" = (/obj/effect/decal/cleanable/robot_debris/up,/turf/open/floor/plasteel/shuttle/yellow,/area/ruin/powered)
-"D" = (/obj/machinery/door/poddoor{id = "mechaship1"; name = "Cargo Bay Door"},/turf/open/floor/plasteel/shuttle/yellow,/area/ruin/powered)
-"E" = (/obj/effect/decal/cleanable/robot_debris,/turf/open/floor/plasteel/shuttle/yellow,/area/ruin/powered)
-"F" = (/obj/structure/mecha_wreckage/durand,/turf/open/floor/plasteel/shuttle/yellow,/area/ruin/powered)
+"v" = (/obj/structure/mecha_wreckage/phazon,/turf/open/floor/plasteel/shuttle/yellow{initial_gas_mix = "TEMP=2.7"},/area/ruin/powered)
+"w" = (/turf/open/floor/plasteel/shuttle/yellow{initial_gas_mix = "TEMP=2.7"},/area/ruin/powered)
```
Comment (xxalpha) [inline _maps/RandomRuins/SpaceRuins/mechtransport.dmm]: ``plasteel/airless/shuttle/yellow`` please

**Comment by optimumtact** [comment]
Without oxygen pr cannoot evoulate?

**Pull request merged**